source/CommonJobs/CommonJobs.Application.EvalForm/Commands/GetEvaluationCalifications.cs

```diff
@@ -0,0 +1,72 @@
+﻿using CommonJobs.Application.EvalForm.DTOs;
+using CommonJobs.Domain.Evaluations;
+using CommonJobs.Infrastructure.RavenDb;
+using Raven.Client.Linq;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+
+namespace CommonJobs.Application.EvalForm.Commands
+{
+    public class GetEvaluationCalifications : Command<CalificationsDTO>
+    {
+        private string _period { get; set; }
+
+        private string _evaluatedUser { get; set; }
+
+        private string _loggedUser { get; set; }
+
+        public GetEvaluationCalifications(string period, string evaluatedUser, string loggedUser)
+        {
+            _period = period;
+
+            _loggedUser = loggedUser;
```

Comment:
I am not sure about the approach of make this command so intelligent. I think that upper layers should take care about what to show depending on current logged user. I am not really sure, but it looks that it will bring some unnecessary complexity.